src/cabanamd_impl.h

```diff
@@ -78,19 +84,15 @@ void CbnMD<t_System, t_Neighbor>::init( InputCL commandline )
     // Read input file
     input->read_file();
     nsteps = input->nsteps;
-    if ( system->do_print )
+    if ( _print_rank )
     {
-        printf( "Read input file\n" );
+        std::cout << "Read input file." << std::endl;
```

Comment:
For runs on machines like summit you may not want to write to `cout` but instead maybe a file. You can pass in a general `std::ostream` to enable this.

Reply:
That's a good point. I'm going to refactor this to use output/error files instead.

Reply:
Done